runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -2,6 +2,7 @@
 
 import javax.annotation.Nonnull;
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Arrays'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)